CONTRIBUTING.md

```diff
@@ -50,6 +50,12 @@ After cloning your fork, follow these steps to set up the project locally,
 4. **Do setup for projects by running:** 
       ```bash
       pnpm setup:project
+5. **In a terminal, start the dependencies (rabbitmq, mongodb and minio) as Docker containers by running the following command:**
+      ```bash
+      docker compose -f docker/dependencies.compose.yml up
+      ```
+    Now you can access the MinIO Console on http://localhost:9001/.
```

Comment:
@Sidd-77 Do you think a developer will be needed to access the console of MinIO? If not, we can remove it.

Reply:
Mentioning `Now you can access the MinIO Console on http://localhost:9001/`